xamarin/Xamarin.Forms
# Setup android to be properly multi targeted

### Description of Change ###

Setup the android projects to use a more sane setup for multitargeting


### Platforms Affected ### 
- Android

### Testing Procedure ###
- Pull down the nugets and make sure they work
- pull down the project on VS Mac and Windows and make sure you don't have any compile issues

### PR Checklist ###
